gulp/tasks/scss.js

```diff
@@ -0,0 +1,53 @@
+import dartSass from "sass";
+import gulpSass from "gulp-sass";
+import rename from "gulp-rename"
+
+import cleanCss from "gulp-clean-css" //Compressing CSS file
+import autoprefixer from "gulp-autoprefixer"; //Adding of vendor prefixes
+import groupCssMediaQueries from "gulp-group-css-media-queries"; //Add media to groups
+
+const sass = gulpSass(dartSass);
+
+export const scss = () => {
+	return app.gulp.src(app.path.src.scss, { sourcemaps: app.isDev })
+		.pipe(app.plugins.plumber(
+			app.plugins.notify.onError({
+				title: "SCSS",
+				message: "Error: <%= error.message %>"
+			})
+		))
+		.pipe(sass({
+			outputStyle: "expanded"
```

Comment:
why do you need this?

Reply:
It gives me expanded CSS in dev mode. What wrong?

Reply:
I am asking about outputStyle: "expanded" setting

Reply:
What changes for you if you just use `sass()`

Reply:
Probably it's expanded by default)). Removed. Done.